src/Microsoft.Diagnostics.ExtensionCommands/DumpAsyncCommand.cs

```diff
@@ -107,6 +107,16 @@ public override void ExtensionInvoke()
                 return;
             }
 
+            ClrStaticField? taskCompletedSentinnelType = taskType.GetStaticFieldByName("s_taskCompletionSentinel");
```

Comment:
We'll want to add a comment on this in dotnet/runtime saying that sos depends on it.  We've done that for other fields to help keep track of what we need to be careful around when refactoring.

Reply:
I had a branch doing this. I'll open the PR soon.